lib/stream-chat/client.rb

```diff
@@ -54,6 +54,19 @@ def initialize(api_key = '', api_secret = '', timeout = 6.0, **options)
       end
     end
 
+    # initializes a Stream Chat API Client from STREAM_KEY and STREAM_SECRET
+    # environmental variables.
+    def self.from_env
+      Client.new(ENV['STREAM_KEY'], ENV['STREAM_SECRET'])
```

Comment:
As we have `@api_key = api_key || ENV['STREAM_KEY']` do you think we need to provide values here instead of just `Client.new`?

I also see that we need to change `def initialize(api_key = '', api_secret = '', timeout = 6.0, **options)` to `def initialize(api_key = nil, api_secret = nil, timeout = 6.0, **options)` so in function it will pick up environment variables. Seems like a bug.